### ISSUE TITLE
!default flag throws linting error

### ISSUE THREAD

**Issue by flachware** [stylelint-scss/stylelint-scss#649]
Hi,
now that `stylelint-config-recommended` has added the `annotation-no-unknown` rule, the `!default` flag throws an error. Is stylelint-scss going to extend the rule for Sass annotations (https://github.com/stylelint/stylelint/issues/4508#issuecomment-573359354) or should I specify `!default` in the `ignoreAnnotations` secondary option of the rule?

**Comment by xrow** [comment]
I came accross this too. I think it needs fixing.

**Comment by shirotech** [comment]
Comparing not equals is also throwing error:

```scss
if($current-value != $value)
```

```
Unexpected unknown annotation "!="  annotation-no-unknown
```

**Comment by XhmikosR** [comment]
This was reverted so it should not happen with the latest version.

**Comment by flachware** [comment]
Looks good to me.

**Comment by jdelStrother** [comment]
I'm still seeing this with stylelint 15.3.0 and stylelint-scss 4.6.0. What am I missing?

```shell
$ yarn stylelint 'app/**/*.scss'                                                                                                                                                                                                            
app/assets/stylesheets/_definitions.scss
 47:16  ✖  Unexpected unknown annotation "!default"  annotation-no-unknown
 77:24  ✖  Unexpected unknown annotation "!default"  annotation-no-unknown
 78:20  ✖  Unexpected unknown annotation "!default"  annotation-no-unknown
```

That's from, eg,:

```scss
$blue: #00a2f1 !default;
```
